app/models/street/StreetEdgeTable.scala

```diff
@@ -159,9 +159,19 @@ object StreetEdgeTable {
         case _ => completedAuditTasks
       }
 
+      val filteredTasks = if (highQualityOnly) {
+        for {
+            tasks <- auditTaskQuery
+            stats <- UserStatTable.userStats if tasks.userId === stats.userId
+            if stats.highQuality && stats.excludeManual.isEmpty
```

Comment:
Note that here you used
```
if stats.highQuality && stats.excludeManual.isEmpty
```

But for the other function you used
```
if stats.highQuality && (stats.excludeManual.isEmpty || !stats.excludeManual)
```

The latter should be used in both places!